tx/textual/valuerenderer/coins.go

```diff
@@ -36,6 +41,10 @@ func (vr coinsValueRenderer) Format(ctx context.Context, v protoreflect.Value) (
 	// If it's a repeated Coin:
 	case protoreflect.List:
 		{
+			if protoCoins.Len() == 0 {
+				return []Screen{{Text: emptyCoins}}, nil
+			}
```

Comment:
Thanks for reverting core/coins/format.go. However, I think these 3 lines should go in the Format function there